serverless.yml

```diff
@@ -1,12 +1,16 @@
 # For full config options, check the docs:
 #    docs.serverless.com
 
-service: projectworks-leave-notifications
+# the service name is used as a prefix for infrastructure created via
+# CloudFormation. Some infrastructure has limits on name length so don't make
+# the service name too long
+service: projectworks-leave-notify
 frameworkVersion: '2'
 provider:
   name: aws
   region: ap-southeast-2
   runtime: nodejs12.x
+  lambdaHashingVersion: "20201221" # this will be default in serverless v3
```

Comment:
This silences a deprecation warning and should allow us an easier upgrade to Serverless v3